src/routes/place.js

```diff
@@ -17,7 +17,19 @@ const placeRoute = {
       authMiddleware(),
       validate(placeValidation.locatePlaces),
       placeController.locatePlaces,
-    )
+    ),
+    router.put(
+      "/:id/:section/checkin",
```

Comment:
Quizás sería mejor hacer un router aparte para las secciones, que sea un subrouter del de places, y que tenga los siguientes endpoints:
- `GET /places/:placeId/sections` para obtener todas las secciones del lugar
- `PUT /places/:placeId/sections/:sectionId/checkin` para hacer check in a la sección
- `PUT /places/:placeId/sections/:sectionId/checkout` para hacer check out de la sección

Reply:
Por ahora los dejé en el mismo router para no alargarlo.
Como se me complica probar no quiero romper nada jaja
(voy a ver como armar una bd local)